tests/swarm-deploy/test_service_restart.py

```diff
@@ -70,17 +73,29 @@ def deployed_simcore_stack(
     return core_stack_services
 
 
+SERVICES_AND_EXIT_CODES = [
+    ("webserver", 0),
+    ("static-webserver", 15),

```

Comment:
15? why is that and what does it mean?

Reply:
Yeah, this is an odd one. When this service is shut down, because we are doing a reboot, it exists with exit code 15. I was not able to trace why. But this is the usual result. Since this is and external service, we do not have any way of controlling the exit code.